libparsec/crates/testbed/src/template/crc_hash.rs

```diff
@@ -73,20 +73,24 @@ impl<T: CrcHash> CrcHash for Vec<T> {
     }
 }
 
-impl<A: CrcHash, B: CrcHash> CrcHash for HashMap<A, B> {
+impl<A: CrcHash + std::cmp::Ord, B: CrcHash> CrcHash for HashMap<A, B> {
     fn crc_hash(&self, hasher: &mut crc32fast::Hasher) {
         hasher.update(b"HashMap");
-        for (k, v) in self.iter() {
+        let mut items: Vec<_> = self.iter().collect();
+        items.sort_unstable_by(|(a, _), (b, _)| a.partial_cmp(b).unwrap());
+        for (k, v) in items {
```

Comment:
Could you use [Itertool::sorted_unstable_by](https://docs.rs/itertools/0.11.0/itertools/trait.Itertools.html#method.sorted_unstable_by) ?

Reply:
This is pretty simple code, so I don't see the need to use yet another dependency

(and the [itertool version just do this under the hood anyway](https://docs.rs/itertools/0.11.0/src/itertools/lib.rs.html#2709-2711))

Reply:
We already have itertool has a dependency